src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private final String color = ColorSupplier.getRandomColor();
```

Comment:
```suggestion
    private final String color;
```
let's initialize this field in the constructor